client/payment-methods-map.js

```diff
@@ -240,3 +240,17 @@ export default {
 		capability: 'cashapp_payments',
 	},
 };
+
+// Enable Bacs according to feature flag value
+if ( window.wc_stripe_settings_params?.is_bacs_enabled ) {
+	paymentMethodsMap.bacs_debit = {
+		id: 'bacs_debit',
+		label: 'Bacs Direct Debit',
+		description:
+			'Bacs Direct Debit enables customers in the UK to pay by providing their bank account details.',
+		Icon: icons.card,
```

Comment:
Temporary icon. It will change once we have an icon defined. Same for the label and description.